packages/dashboard/src/components/schedule-visualizer/tests/robot-trajectories-overlay.test.tsx

```diff
@@ -87,8 +55,23 @@ describe('RobotTrajectoriesOverlay', () => {
   });
 
   it('renders without crashing', async () => {
-    const wrapper = await createWrapper(mapBound, trajectoryConflict, robots, [trajectoryValue]);
-    expect(toJson(wrapper)).toMatchSnapshot();
-    wrapper.unmount();
+    const settings = defaultSettings();
+    let root: RenderResult;
+    await waitFor(() => {
+      root = render(
+        <LMap>
+          <SettingsContext.Provider value={settings}>
+            <RobotTrajectoriesOverlay
+              bounds={mapBound}
+              robots={robots}
+              trajectories={[trajectoryValue]}
+              conflicts={trajectoryConflict}
+            />
+          </SettingsContext.Provider>
+        </LMap>,
+      );
+      expect(root.queryAllByTestId('trajMarker')).toBeTruthy();
```

Comment:
Hi @vallq one last thing, this is the same thing that I explained before if([]) is `true`. So I added that in my version, and it hangs the test. So, that assertion it's not working. Can you try it on your pc? If it's not working for you too, maybe it is not the way of doing things. So can you dig a bit about it. If you can find a solution, we can use it as a smoke test (renders without an assertion).

Reply:
hey @matiasbavera, 
I've not been able to get a solution, hence I've removed the expect line for now. what are your thoughts?

Reply:
Ok, let's merge this PR, and can you create an issue on this? detailing what you want to achieve (trying to select an element inside a rendered component) and what is happening when it fails. Also what you tried to solve this issue.

Reply:
Maybe we can solve this by rendering some part of the component first, but I'm not sure. Let's discussed it in the issue.

Reply:
Alright! thanks @matiasbavera